ember_debug/object-inspector.js

```diff
@@ -1,10 +1,11 @@
 import PortMixin from 'ember-debug/mixins/port-mixin';
 import { compareVersion } from 'ember-debug/utils/version';
 import { isComputed, isDescriptor, getDescriptorFor } from 'ember-debug/utils/type-check';
+import { typeOf } from "./utils/type-check";
```

Comment:
Single quotes. First time I didn't want to reference all the instances but there's at least another one more that needs to be updated.

Reply:
:(  I though i catched them all...